lib/rack/deflater.rb

```diff
@@ -22,7 +22,7 @@ class Deflater
     # [app] rack app instance
     # [options] hash of deflater options, i.e.
     #           'if' - a lambda enabling / disabling deflation based on returned boolean value
-    #                  e.g use Rack::Deflater, :if => lambda { |env, status, headers, body| body.length > 512 }
+    #                  e.g use Rack::Deflater, :if => lambda { |*, body| body.map(&:bytesize).reduce(0, :+) > 512 }
```

Comment:
Could be

```ruby
if: ->(*, body) { body.map(&:bytesize).reduce(0, :+) > 512 }
```

which would [please](https://github.com/bbatsov/ruby-style-guide#lambda-multi-line) the [👮](https://github.com/bbatsov/rubocop).

Should be fine as rack now require Ruby 2.2.2.